### PR TITLE
Add actions with posts

### PR DESCRIPTION
**Pull request by PetrushynskyiOleksii**
**Ref #11** 

1. Review specific post
2. Edit post
3. Add post

### CODE REVIEW

File: blog/templates/blog/base.html [hunk 1]
```diff
@@ -12,7 +12,12 @@
 
   <body>
     <div class="page-header">
-        <h1><a href="/">Django Girls Blog</a></h1>
+      {% if user.is_authenticated %}
+      <a href="{% url 'post_new' %}" class="top-menu">
+        <span class="glyphicon glyphicon-plus"></span>
+      </a>
+      {% endif %}
+      <h1><a href="/">Django Girls Blog</a></h1>
```
Comment (webknjaz) [inline blog/templates/blog/base.html]: Could you please integrate i18n (`{% trans %}`/gettext) into all strings in your app?

File: blog/forms.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+"""Forms of blog app."""
+
+from django import forms
+
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Form for Post model."""
```
Comment (webknjaz) [inline blog/forms.py]: Forms are not for models, but for some logic.

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Review by webknjaz — COMMENTED** [review]
Shall I expect any tests here as well?

File: blog/templates/blog/base.html (hunk 1 above)
Comment (PetrushynskyiOleksii) [reply blog/templates/blog/base.html]: @webknjaz done, but I dont know if it is correct

File: blog/forms.py (hunk 2 above)
Comment (PetrushynskyiOleksii) [reply blog/forms.py]: Fixed.

File: blog/locale/en/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (webknjaz) [inline blog/locale/en/LC_MESSAGES/django.po:1]: You don't have to generate this file just yet: for English, original values will be used automatically.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**